### ISSUE TITLE
Error happens when importing web-ifc-api-node

### ISSUE THREAD

**Issue by posutsai** [tomvandig/web-ifc#105]
Hi, I am new to web-ifc. Thanks for your great job. Following the [hellowold example](https://github.com/IFCjs/hello-world/tree/main/examples/web-ifc/ifc-to-json/node) which convert IFC file into JSON with nodejs, I meet an error when `require('web-ifc/web-ifc-api-node');`. I am using version 0.0.29. Here's the error message.

```
if (self.crossOriginIsolated) {
^

ReferenceError: self is not defined
    at Object.<anonymous> (/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:53611:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/ubuntu/git/ifc-reader/node/main.js:2:16)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
```
Could you give me some advice how to solve the issue?

**Comment by maciejlutostanski** [comment]
In node.js, you don't have self. [node-self](https://github.com/leegeunhyeok/node-self#readme) is your friend here.

**Comment by tomvandig** [comment]
Hi guys, sorry about this, working on the multi threading is a bit cumbersome. I’ll fix it when I have time. In the meantime consider rolling back a couple versions, that should work.

**Comment by posutsai** [comment]
Is there any suggested version? I've try various past versions (0.0.25 ~ 0.0.28) with the sample [IFC Schependomlaan.ifc](https://github.com/buildingSMART/Sample-Test-Files/blob/master/IFC%202x3/Schependomlaan/Design%20model%20IFC/IFC%20Schependomlaan.ifc) but none of these successfully convert the ifc file into json. Here's several error logs of each version.

```
# 0.0.28
/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:53611
if (crossOriginIsolated) {
^

ReferenceError: crossOriginIsolated is not defined
    at Object.<anonymous> (/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:53611:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/ubuntu/git/ifc-reader/node/main.js:2:16)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10) 
```
```
# 0.0.26
TypeError: Cannot convert "undefined" to unsigned int
    at Object.toWireType (/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:4147:21)
    at Object.GetLine (eval at new_ (/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:3846:31), <anonymous>:8:26)
    at IfcAPI.GetRawLineData (/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:47630:28)
    at IfcAPI.GetLine (/home/ubuntu/git/ifc-reader/node_modules/web-ifc/web-ifc-api-node.js:47579:28)
    at saveProperties (/home/ubuntu/git/ifc-reader/node/main.js:42:26)
    at getAllItemsFromLines (/home/ubuntu/git/ifc-reader/node/main.js:33:13)
    at GetAllItems (/home/ubuntu/git/ifc-reader/node/main.js:26:5)
    at LoadFile (/home/ubuntu/git/ifc-reader/node/main.js:12:22)
 ```
 
 
Looks like 0.0.26 actually pass the if-condition which was failed. Is there any clue how I can move further? Thank you.

**Comment by tomvandig** [comment]
PR merged, should be fixed in 0.0.30. If problems remain, please reopen the issue.
